runtime/queries/zig/highlights.scm

```diff
@@ -125,44 +119,53 @@ field_constant: (IDENTIFIER) @constant
   "or"
   "and"
   "orelse"
-] @operator
+] @keyword.operator
 
 [
   "struct"
   "enum"
   "union"
-  "error"
   "packed"
   "opaque"
-] @keyword
+  "export"
+  "extern"
+  "linksection"
+] @keyword.storage.type
+
+[
+  "const"
+  "var"
+  "threadlocal"
+  "allowzero"
+  "volatile"
+  "align"
+] @keyword.storage.modifier
 
 [
   "try"
   "error"
   "catch"
-] @function.macro
+] @keyword.control.exception
 
-; VarDecl
 [
-  "threadlocal"
   "fn"
 ] @keyword.function
 
 [
-  "const"
-  "var"
   "test"
+] @keyword
+
+[
   "pub"
   "usingnamespace"
-] @keyword
+] @namespace
```

Comment:
typically `@namespace` is for the namespace names themselves. `pub` is `@keyword` in the rust highlights. `usingnamespace` might be better as `@keyword.control.import` although I'm not familiar with Zig so I'm not sure that's appropriate

Reply:
I was not sure about it either, but Your suggestion makes sense. However I would leave `pub` also in `@keyword.control.import` as its a keyword for controlling what can/can not be imported.